tips/033_tip_dynamic_env_variables.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: default
+title: Dynamic environement variables in the front end
+sitemap:
+priority: 0.1
+lastmod: 2020-07-01T10:50:00-00:00
+---
+# Dynamic environement variables in the front end
+
+__Tip submitted by [@yelhouti](https://github.com/yelhouti)__
+
+Let's say you need to update the value of a variable in the front end after your code have been compiled. (ex: your IdP Url, the email to use for contacts...)
+
+One way is to have it as part of your application.yml and have the backend return it to the front using a new custom endpoint, the same way we do in: `AuthInfoResource.java` when using OAuth2.
+
+Another way that removes the need for this endpoint and offers better flexibility and less code is to have a new file called `env.js` that looks like this:
+
+```
```

Comment:
Is it possible in Jekyll to enable syntax highlighting for code blocks?

Reply:
Aucune idée, jamais utilisé Jekyll, dit moi quoi faire et je le fais sans problème.

Reply:
You can specify the language next to the ``` like [here](https://github.com/jhipster/jhipster.github.io/commit/92195598afecdd1b0675a98abbfa8c96351646f6#diff-8f89e32c752d5048d1aafce1d01f05b0R71) for example.  [Supported languages](https://github.com/rouge-ruby/rouge/wiki/List-of-supported-languages-and-lexers) include typescript and javascript